pyro/distributions/transforms/__init__.py

```diff
@@ -171,7 +165,6 @@ def iterated(repeats, base_fn, *args, **kwargs):
     "ConditionalRadial",
     "ConditionalSpline",
     "ConditionalSplineAutoregressive",
-    "CorrLCholeskyTransform",
```

Comment:
Shouldn't the new CorrCholeskyTransform apper in this list?

Reply:
It should be added with other PyTorch transforms below:

```
__all__.extend(torch_transforms)
```